shared/actions/chat.js

```diff
@@ -338,6 +350,29 @@ function _unboxedToMessage (message: MessageUnboxed, idx: number, yourName): Mes
   }
 }
 
+function * _startConversation (action: StartConversation): SagaGenerator<any, any> {
```

Comment:
Start a conversation and reload inbox/select the conversation. This seems wonky due to some core issues so you have to click 'start chat' twice sometimes. When it switches to the chat again it can be slow due to a ton of identifies and etc. if you let it sit it'll work though